src/components/profile/Dashboard.jsx

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { connect } from "react-redux";
+import { Link } from "react-router-dom";
+
+import { getCurrentProfile } from "../../redux/actions/profileActions";
+
+class Dashboard extends Component {
+  componentDidMount() {
+    const { getProfile } = this.props;
+    getProfile();
```

Comment:
I think `this.props.getProfile()` is enough here as it's only being used in one place..no need to destructure

Reply:
@kimotho-njoki If I don't destructure eslint is complaining. I had initially implemented it without destructuring.